pom.xml

```diff
@@ -393,6 +395,7 @@
               <exclude>src/ports/greenplum/cmake/FindGreenplum_4_2.cmake</exclude>
               <exclude>src/ports/greenplum/cmake/FindGreenplum_4_3.cmake</exclude>
               <exclude>src/ports/greenplum/cmake/FindGreenplum_4_3ORCA.cmake</exclude>
+              <exclude>src/ports/greenplum/cmake/FindGreenplum_5.cmake</exclude>
```

Comment:
Curious, I thought I did add it in one of the prior commits. I'll add it during the merge.